volatility3/framework/__init__.py

```diff
@@ -216,9 +217,20 @@ def _zipwalk(path: str):
 def list_plugins() -> Dict[str, Type[interfaces.plugins.PluginInterface]]:
     plugin_list = {}
     for plugin in class_subclasses(interfaces.plugins.PluginInterface):
-        plugin_name = plugin.__module__ + "." + plugin.__name__
+        # If the name of the plugin equals the last part of the module path, we omit the name of the
```

Comment:
I really just don't see this as necessary.  The plugin's name *is* the complete value, and it really doesn't hurt a lot.  Convention has led almost every example to be the same module name as plugin name, but there's no real reason for that and we need to cater that changing at some point, so this bit shouldn't go in either.